src/io/flutter/view/FlutterViewMessages.java

```diff
@@ -39,9 +39,8 @@ public static void sendDebugActive(@NotNull Project project,
                                      @NotNull VmService vmService) {
     final MessageBus bus = project.getMessageBus();
     final FlutterDebugNotifier publisher = bus.syncPublisher(FLUTTER_DEBUG_TOPIC);
-    app.setVmService(vmService);
     assert(app.getFlutterDebugProcess() != null);
-    app.setPerfService(new PerfService(app.getFlutterDebugProcess(), vmService));
+    app.setVmServices(vmService, new PerfService(app.getFlutterDebugProcess(), vmService));
```

Comment:
Add a TODO to rename PerfService to VMService.

Reply:
done